compiler/rustc_codegen_cranelift/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+#![cfg_attr(not(bootstrap), allow(internal_features))]
+#![cfg_attr(not(bootstrap), feature(rustdoc_internals))]
+#![cfg_attr(not(bootstrap), doc(rust_logo))]
```

Comment:
Please see

```rust
// Note: please avoid adding other feature gates where possible
```

below. I recently added `-Zallow-features=rustc_private` to cg_clif's build system to avoid extra features like this, but that change hasn't been synced back to this repo yet. But now that I am trying to sync to cg_clif's repo it causes a build error. I had to add an extra condition on the `doc` cfg to fix this: https://github.com/bjorn3/rustc_codegen_cranelift/commit/bdc597d512b080ddc7bb8c1eef938e002686243e.